persist_error/handler.py

```diff
@@ -46,11 +46,21 @@ def lambda_handler(event, context):
         # abort retry attempts if there more failures than the set condition
         # send a message to SNS for human to deal with it
         subject = 'Excessive Capture Failures Reported'
+
+        try:
+            json_file = initial_input['Record']['s3']['object']['key']
+        except KeyError:
+            json_file = 'could not parse json file from state machine input'
```

Comment:
should we append initial_input to json_file in the hope it may give a clue if something is malformed?